config/bundles.php

```diff
@@ -1,20 +1,10 @@
 <?php
 
-/*
- * This file is part of the Secret Santa project.
- *
- * (c) JoliCode <coucou@jolicode.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
```

Comment:
Why?

Reply:
The file was re-written when I removed web-server-bundle, I ran cs-fixer to add it back.